teensy/main/main.ino

```diff
@@ -140,5 +176,22 @@ void print_all_sensors() {
   Serial.println(count);
   count++;
   Serial1.println();
+  
+  output.concat("^Count: ");
+  output.concat(count);
+
```

Comment:
Can you add a newline to the end of the string so that each data frame is on its own line?